src/views/TechnicalAssistance/RequestForm/index.tsx

```diff
@@ -0,0 +1,297 @@
+
+  const { id, step, view } = useParams<{
+    /** Request id */
+    id: string;
+    /** Form step slug */
+    step?: string;
+    /** Form step subview */
+    view?: string;
+  }>();
+
+  const [create, createResult] = useMutation<
+    CreateTrbRequest,
+    CreateTrbRequestVariables
+  >(CreateTrbRequestQuery);
+
+  const [get, getResult] = useLazyQuery<GetTrbRequest, GetTrbRequestVariables>(
+    GetTrbRequestQuery
+  );
+
+  // const [update, updateResult] = useMutation<
+  //   UpdateTrbRequest,
+  //   UpdateTrbRequestVariables
+  // >(UpdateTrbRequestQuery);
+  // todo update, updateResult
+
+  // Assign request data from the first available query response
+  // Prioritize by latest type: update, get, create
+  // eslint-disable-next-line camelcase
+  const request: CreateTrbRequest_createTRBRequest | undefined =
+    getResult.data?.trbRequest || createResult.data?.createTRBRequest;
+  // const data = useMemo(() => {
+  //   if (getResult.data) {
+  //     return getResult.data.trbRequest;
+  //   }
+  //   if (createResult.data) {
+  //     return createResult.data.createTRBRequest;
+  //   }
+  //   return undefined;
+  // }, [getResult, createResult]);
+  // console.log('data', request);
+
+  useEffect(() => {
+    // Create a new request if `id` is new and go to it
+    if (id === 'new') {
+      // Request type must be defined for new requests
+      if (requestType) {
+        if (!createResult.called) {
+          // Create the new request
+          create({ variables: { requestType } }).then(res => {
+            // Update the url with the request id
+            if (res.data) {
+              history.replace(`/trb/requests/${res.data.createTRBRequest.id}`);
+            }
+          });
+        }
+      }
+      // Redirect to the start if there's no request type
+      else history.replace('/trb/start');
+    }
+    // Fetch request data if not new
+    else if (!request && !createResult.called && !getResult.called) {
+      get({ variables: { id } });
+    }
+    // Get or create request was successful
+    // Continue any other effects with request data
+    else if (request) {
+      // Check step param, redirect to the first step if invalid
+      if (!step || !formSteps.includes(step)) {
+        history.replace(`/trb/requests/${id}/${formSteps[0]}`);
+      }
+    }
+  }, [
+    create,
+    createResult,
+    get,
+    getResult,
+    history,
+    id,
+    request,
+    requestType,
+    step
+  ]);
+
+  const defaultBreadcrumbs = useMemo(
+    () => (
+      <Breadcrumbs
+        items={[
+          { text: t('heading'), url: '/trb' },
+          { text: t('breadcrumbs.taskList'), url: '/trb/task-list' },
+          { text: t('requestForm.heading') }
+        ]}
+      />
+    ),
+    [t]
+  );
+
+  if (getResult.loading || createResult.loading) {
+    return <PageLoading />;
+  }
+
+  if (!step || getResult.error || createResult.error) {
+    return <NotFoundPartial />;
```

Comment:
This `NotFoundPartial` renders outside the Maincontent/GridContainer and takes up the whole width of the screen.  I think it should be contained.  Same for Loading, but less noticeable there. 

Per the previous comment threads, still torn on this rendering here, as the loading flicker is quite noticeable